.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+save-prefix false
```

Comment:
I'll also add strict versioning to `.npmrc` so we are consistent. npm 5.x looks pretty cool, maybe we wouldn't need yarn anymore :D